src/open_inwoner/components/templates/components/UserFeed/UserFeed.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n icon_tags %}
+
+{# if userfeed data is true then show component #}
+<section class="userfeed">
+    <h2 class="h2 {# if userfeed true #}indicator{# endif #}">
+        {% trans "Aanvragen updates" %}
+    </h2>
+    <div class="userfeed__summery">
+        <ul class="userfeed__list">
+            <li class="userfeed__list-item">
+                <p class="userfeed__list-paragraph">
+                    Bij <span>X</span> aanvraag{# if statuses > 1 #}aanvragen{# endif #} is uw actie nodig
```

Comment:
Perhaps:
```
Bij <span>{# num statuses #}</span> aanvraag{# if statuses > 1 #}en{# endif #} is uw actie nodig
```

Reply:
:-) This whole line will probably be completely replaced later on by the back-end (where the plural/single situation might also be handled from there) so this is not definite. I could just remove this and make it completely static.